ddtrace/internal/utils/http.py

```diff
@@ -190,3 +193,8 @@ def w3c_get_dd_list_member(context):
                 log.debug("tracestate would exceed 256 char limit with tag: %s. Tag will not be added.", next_tag)
 
     return ";".join(tags)
+
+
+def w3c_encode_equals(tag_value):
```

Comment:
Would it make sense to create helper that handles all the encoding (so regex sub + converting the `=` character)? Would this simplify `w3c_get_dd_list_member` function? 
We can maybe do the same with the the decode tag logic too.
 
 
```suggestion
def w3c_encode_tag(tag_value):
```